pandunia-lekse.csv

```diff
@@ -2599,6 +2632,7 @@ senegalia|dex|SN||Senegal||Senegal||||||||||||||Senegal|Senegal
 sensa||||sense (detect)||sentir (detectar)|||||||||||||||
 sense||||sense||sentido|||||||||||||||zmysł
 senser||||sensor (detector)|||||||||||||||||
+sensia||||sensation (feeling)|||||||||||||||||
```

Comment:
me fikra "sense" si "a sensation" e "sensia" si "the senses".

Reply:
le unisa "sense" e "sensation". we okey?